Rakefile

```diff
@@ -17,3 +17,14 @@ require "hammer_cli_foreman_discovery/version"
 require "hammer_cli_foreman_discovery/i18n"
 require "hammer_cli/i18n/find_task"
 HammerCLI::I18n::FindTask.define(HammerCLIForemanDiscovery::I18n::LocaleDomain.new, HammerCLIForemanDiscovery.version.to_s)
+
+begin
```

Comment:
I prefer the pattern in https://github.com/theforeman/hammer-cli-foreman-leapp/pull/7/files#diff-ee98e028c59b193d58fde56ab4daf54d43c486ae674e63d50ddf300b07943e0f